core/trino-main/src/test/java/io/trino/sql/planner/optimizations/TestDeterminePartitionCount.java

```diff
@@ -197,10 +200,10 @@ public void testPlanWhenCrossJoinIsScalar()
                         join(INNER, builder -> builder
                                 .right(
                                         exchange(LOCAL,
-                                                exchange(REMOTE, Optional.of(15),
+                                                exchange(REMOTE, REPLICATE,
```

Comment:
This test case is updated to indicate that no partitioning should be selected because only `REPARTITION` exchanges are targeted in the latest revision per your other comment.